util/db/dbutil.go

```diff
@@ -170,7 +172,7 @@ func Retry(fn func() error) (err error) {
 // getDecoratedLogger retruns a decorated logger that includes the readonly true/false, caller and extra fields.
 func (db *Accessor) getDecoratedLogger(fn idemFn, extras ...interface{}) logging.Logger {
 	log := db.logger().With("readonly", db.readOnly)
-	_, file, line, ok := runtime.Caller(2)
+	_, file, line, ok := runtime.Caller(3)
```

Comment:
We increase this to 3 since the "depth" is now longer - we have the "Atomic" function calling the "atomic" function.